.github/workflows/run_tests.yml

```diff
@@ -235,12 +235,11 @@ jobs:
       - name: Install go-swagger
         id: install-go-swagger
         run: |
-          sudo apt-get update
-          sudo apt-get install -y apt-transport-https gnupg curl
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/gpg.2F8CB673971B5C9E.key' | sudo apt-key add -
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/config.deb.txt?distro=debian&codename=any-version' | sudo tee /etc/apt/sources.list.d/go-swagger-go-swagger.list
-          sudo apt-get update
-          sudo apt-get install -y swagger
+          dir=$(mktemp -d)
+          git clone https://github.com/go-swagger/go-swagger "$dir" 
+          cd "$dir"
+          git checkout v0.29.0
```

Comment:
We should probably either
- Update this to `v.0.28.0`
- Use a GitHub direct binary download (also with `v.0.28.0`

Reply:
Reverted to v0.28.0